Submitting the first Python assignment after incoporating my code to the

vikramkatju opened the pull request:
given .ipynb file

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
I added python code to the assignment_1.ipynb file in accordance with the assignment questions

## What did you learn from the changes you have made?
I learnt about how to create user defined python functions and about the inbuilt function sorted

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I was thinking of a way to conflate the two functions used in this program into a single function with the is_case_sensitive parameter set to a default value of False.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No particular challenges

## How were these changes tested?
I ran the code on the Jupyter notebook on which I added the python code.

## A reference to a related issue in your repository (if applicable)

## Checklist
- [x] I can confirm that my changes are working as intended

vikramkatju committed:
Submitting the first Python assignment after incoporating my code to the
given .ipynb file

vikramkatju committed:
Python file had not been saved prior to previous push

vikramkatju committed:
Added comments to the python program just above each of the two
functions.

vikramkatju committed:
Adding some additional comments and deleting one empty python code cell.

vikramkatju committed:
Changed the way of adding comments to the functions by using docstrings
instead of #.

pedram-aliniaye-asli commented:
You did a great job here!
Just consider that in part 1 the function should return a boolean value as the result and not just print out a string.
Please go ahead and do the modifications and then let me know. Thanks!

vikramkatju committed:
Making minor modification to Part 1 so that functions returns a Boolean
value (True or False) instead of printing whether the two strings are
anagrams or not anagrams.

vikramkatju commented:
Hi Pedram, I've made the correction. Thank you.
